### PR TITLE
Mv miracles ewostatewos

### PR DESCRIPTION
**Pull request by MassimoVilla**
Last two miracles of ʾEwosṭātewos from BGV-009

### CODE REVIEW

File: new/LIT6366MEDeath.xml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+                        ሊባኖስ፡ ርጡብ። ውእቱኬ፡ ምጽላለ፡ ብዙኃን፡ አሕዛብ። ወአርጋብ። ምሉአነ፡ ጥበብ፡ ዕፁብኬ፡ ውእቱ፡ ዘከመ፡ አዘዞሙ፡ አቡነ፡ ኤዎስጣቴዎስ፡
+                        ለደቂቁ፡
+                    </ab>
+                </div>
+                <div type="textpart" subtype="explicit" xml:lang="gez">
+                    <note>
+                        This explicit is taken from <ref type="mss" corresp="BGV009"/>.</note>
+                    <ab>
+                        ወሶበ፡ ሞቱ፡ እሙንቱ፡ አናብስት፡ ቀበርዎሙ፡ ከመ፡ ሰብእ፡ በፀጋመ፡ መቃብሪሁ፡ ለአቡነ፡
+                        ኤዎስጣቴዎስ፡ የአምሩ፡ ከመ፡ ነበሩ፡ ምስሌሁ፡ ፲ወ፬ ዓመታት፡ እንዘ፡ ይትለአክዎ። ወይትዔዘዙ፡ ኀበ፡ ኵሉ፡ ዘሀለየ። ወሶበ፡ ርእየ፡
```
Comment (SusanneHummel) [inline new/LIT6366MEDeath.xml:90]: ፲ወ፬ ዓመታት፡
We decided to not place a space after the numeral but this decision is limited to our Mss-entries?

File: new/LIT6366MEDeath.xml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+                        ለደቂቁ፡
+                    </ab>
+                </div>
+                <div type="textpart" subtype="explicit" xml:lang="gez">
+                    <note>
+                        This explicit is taken from <ref type="mss" corresp="BGV009"/>.</note>
+                    <ab>
+                        ወሶበ፡ ሞቱ፡ እሙንቱ፡ አናብስት፡ ቀበርዎሙ፡ ከመ፡ ሰብእ፡ በፀጋመ፡ መቃብሪሁ፡ ለአቡነ፡
+                        ኤዎስጣቴዎስ፡ የአምሩ፡ ከመ፡ ነበሩ፡ ምስሌሁ፡ ፲ወ፬ ዓመታት፡ እንዘ፡ ይትለአክዎ። ወይትዔዘዙ፡ ኀበ፡ ኵሉ፡ ዘሀለየ። ወሶበ፡ ርእየ፡ 
+                        ጣዕዞሙ፡ አእመርዎሙ፡ ሰብእ፡ ከመ፡ አርድእቱ፡ ለአቡነ፡ ኤዎስጣቴዎስ፡ድንግል፡ ሰባኬ፡ ወንጌል፡ ውእቱኬ፡ ጸዋሬ፡ መስቀል፡
```
Comment (SusanneHummel) [inline new/LIT6366MEDeath.xml:91]: ኤዎስጣቴዎስ፡ድንግል፡ 

space is needed

File: new/LIT6366MEDeath.xml [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+                        ዘቃል፡ ከያዴ፡ ማዕበል፡ በኃይለ፡ አማኑኤል፡ ውኵል፡ በጸሎቱ፡
```
Comment (SusanneHummel) [inline new/LIT6366MEDeath.xml:92]: I think something is wrong with ውኵል

**Review by SusanneHummel — COMMENTED** [review]
Thank you Massimo!

File: new/LIT6365MESick.xml [hunk 4]
```diff
@@ -0,0 +1,96 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6365MESick" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Miracle of ʾEwosṭātewos: on the sick who was healed and on his daughter who had a snake in her womb</title>
```
[the remark this answers is not in the document]
Comment (MassimoVilla) [reply new/LIT6365MESick.xml]: Ok, perhaps it is a bit old fashioned. I've updated the titles of the old miracles as well.

File: new/LIT6365MESick.xml [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+                    <ab>
+                        በከመ፡ ይቤ፡ መጽሐፍ፡ ኵሎ፡ ዘሰማዕነ፡ ወዘርአነ፡ ወዘነገሩነ፡ አበዊነ። ወኢኀብኡ፡ እምደቂቆሙ፡ ለካልእ፡ ትውልድ፡ ወነገሩ፡ ስብሐቲሁ፡ ለእግዚአብሔር፡ ኃይሎሂ፡ 
+                        ወመንክሮ፡ ዘገብረ፡ ወካዕበ፡ ይቤ፡ ኢዩኤል፡ ሰምዑ፡ ዘንተ፡ ሊቃውንት።
+                    </ab>
+                </div>
+                <div type="textpart" subtype="explicit" xml:lang="gez">
+                    <note>
+                        This explicit is taken from <ref type="mss" corresp="BGV009"/>.</note>
+                    <ab>
+                        ናሁኬ፡ ውእቱ፡ ለአሕዛብ፡ በምግባረ፡ ሠናይ፡ ኮኖሙ፡ አርአያ። ወገብረ፡ መንክረ፡ ወመድምመ፡ በስሙ፡ ለኬንያ። እንዘ፡ ሀለዉ፡ አጽምቲሁ፡ በአርማንያ። እስመ፡
```
[the remark this answers is not in the document]
Comment (MassimoVilla) [reply new/LIT6365MESick.xml]: you are right, thank you

File: new/LIT6366MEDeath.xml (hunk 1 above)
Comment (MassimoVilla) [reply new/LIT6366MEDeath.xml]: I missed this occurrence

**Review by SusanneHummel — APPROVED** [review]

**Comment by antobrita** [comment]
Thank you, Massimo!

**Pull request merged**